django_server/.github/workflows/test.yml

```
@@ -0,0 +1,260 @@
+      with:
+        python-version: ${{ env.PYTHON_VERSION }}
+    
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install safety bandit
+    
+    - name: Run Safety check
+      run: safety check --json
+      continue-on-error: true
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)